test/utils/update/fetch.managing.officers.private.data.spec.ts

```diff
@@ -103,7 +118,7 @@ describe("fetchManagingOfficersPrivateData", () => {
     expect(appData.update?.review_managing_officers_corporate?.length).toBeUndefined();
   });
 
-  test("should not map MO private data if existing MO does not have ch_reference", async () => {
+  test("should not map Individual MO private data if existing MO does not have ch_reference", async () => {
     if (appData.update && appData.update.review_managing_officers_individual) {
```

Comment:
would be better to not mutate appData mock for test order / concurrency

Reply:
This case should really never happen - the review list MOs should always have ch-references. Same for line 136 below.

Reply:
Removed